flocker/ca/functional/test_ca.py

```diff
@@ -29,16 +32,19 @@ def flocker_ca(command, *args):
     """
     command = [EXECUTABLE, command] + list(args)
     try:
-        result = run_process(command)
+        result = run_process(command, **kwargs)
         output = result.output
         status = 0
     except CalledProcessError as e:
+        Message.new(
+            message_type="flocker.ca.functional:ca_initialize_error",
+            error=str(e)).write(Logger())
```

Comment:
Don't need a logger anymore in latest version of Eliot, you can just do `Message.new(...).write()`.